awx/main/management/commands/profile_sql.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand
+
+from awx.main.tasks import profile_sql
+
+
+class Command(BaseCommand):
+    """
+    Enable or disable SQL Profiling across all Python processes.
+    SQL profile data will be recorded at /var/lib/awx/profile/
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('--threshold', dest='threshold', type=float, default=2.0,
+                            help='The minimum query duration in seconds (default=2).  Use 0 to disable.')
+        parser.add_argument('--minutes', dest='minutes', type=float, default=1,
+                            help='How long to record for in minutes (default=1)')
+
+    def handle(self, **options):
+        profile_sql.delay(
```

Comment:
I decided to implement this as a distinct command using memcached and *not* using a `conf.Setting` for a few reasons:

1.  `conf.Setting` *itself* generates queries, and I thought it would be complicated to filter those out.  Also, what if we want to profile slow queries in `conf.settings` code _itself_?
2.  I didn't want the per-query overhead of a setting lookup (even if it's just a memcached hit on most queries, *this* implementation at least memoizes the check to once per second so we're not paying a constant penalty per-query).
3.  A major concern with toggling this in a customer environment is making sure they can _easily_ turn it on for *all* customer nodes, and then make **totally** sure they don't accidentally leave it on somewhere (which could lead to major performance regressions, and _quickly_ fill disk space).  This implementation allows people to _easily_ toggle it on for X minutes and not have to deal with cleanup.